lsb_vsx/build.sh

```diff
@@ -0,0 +1,139 @@
+#!/bin/bash
+
+LSB_VSX_VER="2.0-1"
+LSB_VSX="lsb_vsx-${LSB_VSX_VER}"
+
+PREFIX_LSB_VSX="${PREFIX_PROJECT}/phoenix-rtos-ports/lsb_vsx"
+PREFIX_LSB_VSX_BUILD="${PREFIX_BUILD}/lsb_vsx"
+PREFIX_LSB_VSX_CONFIG="${PREFIX_LSB_VSX}/${LSB_VSX}-config"
+PREFIX_LSB_VSX_MARKERS="$PREFIX_LSB_VSX_BUILD/markers"
+PREFIX_LSB_VSX_FILES="${PREFIX_LSB_VSX_BUILD}/files"
+PREFIX_LSB_VSX_TMP="$PREFIX_LSB_VSX_BUILD/tmp"
+
+b_log "Building lsb-vsx (posix test suite)"
+
+#
+# Download and unpack
+#
+mkdir -p "$PREFIX_LSB_VSX_MARKERS"
+mkdir -p "$PREFIX_LSB_VSX_FILES"
+mkdir -p "$PREFIX_LSB_VSX_TMP"
+mkdir -p "$PREFIX_LSB_VSX_TMP"/tetbin_host
+mkdir -p "$PREFIX_LSB_VSX_TMP"/tetbin_phoenix
+mkdir -p "$PREFIX_LSB_VSX_TMP"/BIN_host
+mkdir -p "$PREFIX_LSB_VSX_TMP"/BIN_phoenix
+
+[ -f "$PREFIX_LSB_VSX/tet_vsxgen_3.02.tgz" ] || wget http://www.opengroup.org/infosrv/lsb/ogdeliverables/LSB-VSX2.0-1/tet_vsxgen_3.02.tgz -P "$PREFIX_LSB_VSX"
+[ -f "$PREFIX_LSB_VSX/lts_vsx-pcts2.0beta2.tgz" ] || wget http://www.opengroup.org/infosrv/lsb/ogdeliverables/LSB-VSX2.0-1/lts_vsx-pcts2.0beta2.tgz -P "$PREFIX_LSB_VSX"
+[ -f "$PREFIX_LSB_VSX/lts_vsx-pcts2.0beta.tgz" ] || wget http://www.opengroup.org/infosrv/lsb/ogdeliverables/LSB-VSX2.0-1/lts_vsx-pcts2.0beta.tgz -P "$PREFIX_LSB_VSX"
+
+#
+# # Store compiler name for later use
+#
+CC_BUF=$CC
+export CC_BUF
+export PREFIX_LSB_VSX
+
+TET_ROOT=${PREFIX_LSB_VSX_BUILD}/files
+export TET_ROOT
+
+#
+# # Install files
+#
+if [ ! -d "$PREFIX_LSB_VSX_FILES"/src ]; then
+	# Find out where to install the test suite
+	if [ ! -d "${TET_ROOT}" ]; then
+	mkdir "${TET_ROOT}"
+	if [ ! -d "$TET_ROOT" ]; then
+		echo "Unable to create installation directory: ${TET_ROOT}"
+		exit 1
+	fi
+	fi
+
+	tet_package=${PREFIX_LSB_VSX}/tet_vsxgen_3.02.tgz
+	echo "Installing the TET and VSXgen test harness"
+	if [ ! -f "$tet_package" ]; then
+	echo "Could not find file \"$tet_package\""
+	echo Aborting
+	exit 1
+	fi
+
+	(cd "$TET_ROOT" && tar xfz "$tet_package")
+	if [ $? -ne 0 ]; then
+	echo "Unable to unpack TET/VSXgen"
+	exit 1
+	fi
+
+	# Install the test suites
+	cd "${PREFIX_LSB_VSX}" || exit
+	test_suites=$(echo lts_*.tgz | sed -e s/\ +//)
+	echo installing test_suites: "$test_suites"
+
+	current_dir=$PWD;
+	for test_suite in $test_suites; do
+	echo "Installing $test_suite"
+	(cd "$TET_ROOT"/test_sets && tar xfz "$current_dir"/"$test_suite")
+	if [ $? -ne 0 ]; then
+		echo "Unable to install test suite: $test_suite"
+	fi
+	done;
+
+	# Configure setup script and profiles
+	sed -e "s@^echo Unconfigured@TET_ROOT=$TET_ROOT@" "$TET_ROOT"/setup.sh.skeleton \
```

Comment:
`-e` should not be needed?
- is this standard `sed` syntax? I know of `"s/to_replace/replace_with" one